napari/_qt/qt_viewer.py

```diff
@@ -296,9 +297,25 @@ def __init__(self, viewer: ViewerModel, show_welcome_screen: bool = False):
         # bind shortcuts stored in settings last.
         self._bind_shortcuts()
 
+        settings = get_settings()
+        self._update_dask_settings(settings.application.dask)
+
+        settings.application.events.dask.connect(self._update_dask_settings)
+
         for layer in self.viewer.layers:
             self._add_layer(layer)
 
+    @staticmethod
+    def _update_dask_settings(dask_setting: DaskSettings = None):
```

Comment:
```suggestion
    def _update_cache_settings(value: Union[bool, float, DaskSettings]):
```

Reply:
But this signature is not the true. 
The code below will not accept bool or float, only DaskSettings or Event.

Reply:
oops.  you're right. It was still working, but I see that it is an Event that is passed in.

Reply:
I'm not sure if we want to specify the type here or not.  If we do, Event will needed to be added to the imports.